### PR TITLE
Improve pmm readme documentation

### PR DESCRIPTION
**Pull request by edithturn**
Improving PMM Readme

1. Added comments to the docker commands, I think it will be better to let the user know what these commands are doing in general and how to open the dashboard.

2. CLA link fixed, it was giving "the page does not exist" so, I added this **https://cla-assistant.percona.com/percona/grafana-dashboards** (correct me if this link is wrong) instead of **https://cla-assistant.percona.com/<linktoCLA>**

Any feedback is appreciated!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ Before submitting code or documentation contributions, you should first complete
 
 ### 1. Sign the CLA
 
-Before you can contribute, we kindly ask you to sign our [Contributor License Agreement](https://cla-assistant.percona.com/<linktoCLA>) (CLA). You can do this using your GitHub account and one click.
+Before you can contribute, we kindly ask you to sign our [Contributor License Agreement](https://cla-assistant.percona.com/percona/grafana-dashboards) (CLA). You can do this using your GitHub account and one click.
```
Comment (BupycHuk) [inline CONTRIBUTING.md:102]:
```suggestion
Before you can contribute, we kindly ask you to sign our [Contributor License Agreement](https://cla-assistant.percona.com/percona/pmm) (CLA). You can do this using your GitHub account and one click.
```

**Review by BupycHuk — COMMENTED** [review]
LGTM, suggested a right link

File: CONTRIBUTING.md (hunk 1 above)
Comment (edithturn) [reply CONTRIBUTING.md:102]: Thank you @BupycHuk 💃🏼

**Review by BupycHuk — APPROVED** [review]

**Comment by edithturn** [comment]
Thank you @BupycHuk. Should I make a rebase? 
Please let me know if you have any other feedback about the documentation I added.

**Comment by BupycHuk** [comment]
We need 2 approvals to merge PR, so I requested one more review

**Review by gen1us2k — APPROVED** [review]

**Review by BupycHuk — APPROVED** [review]

**Comment by edithturn** [comment]
Wohoo!! 💃🏼 🥳 🎈 
Thank you @BupycHuk and @gen1us2k

**Comment by BupycHuk** [comment]
@edithturn Thank you for your contribution! :)

**Comment by edithturn** [comment]
@BupycHuk  I think it failed
Screen Shot 2022-07-26 at 08 23 56

**Comment by edithturn** [comment]
wohoo! Looking to contribute more! 🐬

**Pull request merged**